src/thunderbots/software/ai/world/team.cpp

```diff
@@ -83,8 +83,9 @@ void Team::removeExpiredRobots(const Timestamp& timestamp)
         Duration time_diff = timestamp - it->lastUpdateTimestamp();
         if (time_diff.getSeconds() < 0)
         {
-            throw std::invalid_argument(
-                "Error: tried to remove a robot at a negative time");
+            LOG(WARNING) << "Error: tried to remove a robot at a negative time"
+                         << std::endl;
```

Comment:
We don't need to use an `endl` with `LOG`

Reply:
fixed